src/test/java/tests/GmailTest.java

```diff
@@ -0,0 +1,107 @@
+package tests;
+
+import by.issoft.config.WebDriverSingleton;
+import by.issoft.po.*;
+import org.testng.annotations.*;
+
+import java.net.MalformedURLException;
+
+import static org.testng.Assert.assertTrue;
+
+@Listeners(Listener.class)
+class GmailTest {
```

Comment:
Common issue: tests must end with assertion. If you are adding some steps after assertion - they will be not executed if test fails. Move this logic into after method hooks. Let me know if you need additional clarifications from me